update_image.pl

```diff
@@ -375,7 +387,11 @@ sub processConvergedSections {
     {
         if(-e $ocmbfw_original_filename)
         {
-            #Verify image validity
+            #Add header to delivered image
+            my $date = `date`;
+            run_command("$hb_image_dir/pkgOcmbFw.pl --unpackagedBin $ocmbfw_original_filename --packagedBin $ocmbfw_original_filename --timestamp \"$date\" --vendorVersion \"$ocmbfw_version\" --vendorUrl \"$ocmbfw_url\"");
```

Comment:
Would it be better to keep the un-altered file around instead of overwriting it?

Reply:
Yeah, I think so. I have pushed some updates that keep both flows consistent and keep the original file around.